docs/src/content/docs/classifiers/built-in/bedrock-classifier.mdx

```diff
@@ -73,7 +74,8 @@ You can customize the BedrockClassifier by creating an instance with specific op
             'maxTokens': 500,
             'temperature': 0.7,
             'topP': 0.9
-        }
+        },
+        aws_profile_name="dev"
```

Comment:
Change from "dev" to "default" since the profile name by default is "default" =)